### ISSUE TITLE
Character Card Shown

### ISSUE THREAD

**Issue by Nephanor-of-Fraal** [Cohee1207/SillyTavern#94]
A simple option to display the full character card in desktop mode, similar to the character expressions, but using only one image (either the character card, or a custom one) May be easier to implement with the expressions extension.

**Comment by Nephanor-of-Fraal** [comment]
The basic idea is that if you don't have sprites for the expressions extension, it will by default use the full character card image in the spot where the sprites would appear.  This way characters without expressions still are shown to the player in the same way.

**Comment by RossAscends** [comment]
currently we have default expression images serve this purpose. 

simply having the character card image on screen at all times (which would be huge in waifu mode) is not a desirable feature from my point of view.

if you wish to achieve this, you can simply copy the character card into the proper folder and rename it to whatever emotion you want it to display for.

**Comment by Mu2u** [comment]
the character base image from the card should be a universal default, it doesn't make sense to just have the small non clickable version that's just a reference profile pic.  even one image in a large format can add more immersion.

**Comment by Nephanor-of-Fraal** [comment]
Now that would be a hugely awesome feature.  If it understands if it has the default or 28 emotion version, that would be perfect.

**Comment by Mu2u** [comment]
Agree, instead of filling with emoticons it should fill with either the neutral image or the card itself.  I personally don't see an issue with it being the sprite, it can be turned off and on so if it doesn't look right that's fine.  its all optional anyways

**Comment by baphilia** [comment]
@Mu2u  @Nephanor-of-Fraal 
ChatGPT and I whipped up a [couple of simple scripts](https://github.com/baphilia/SillyTavernAutoExpressions) as a workaround.

**Comment by Nephanor-of-Fraal** [comment]
AWESOME!  Thanks!
